files/workflows/ZWEWRF05.xml

```diff
@@ -0,0 +1,215 @@
+        <variableValue name="KEYSTORE_DIRECTORY" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="KEYSTORE_ALIAS" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="ZOWE_USER_ID" scope="instance" noPromptIfSet="false" required="true"/>
+        <variableValue name="ZOWE_GROUP_ID" scope="instance" noPromptIfSet="false" required="true"/>
+		<instructions substitution="false">Run this step to initialize variable values</instructions>
+		<weight>1</weight>
+		<autoEnable>false</autoEnable>
+		<canMarkAsFailed>false</canMarkAsFailed>
+	</step>
+    <step name="Substitute_vars_in_env_file" optional="true">
+        <title>New Custom zowe-setup-certificates.env</title>
+        <description>Creates new zowe-setup-certificates.env in user specified location adn substitutes values</description>
+        <prereqStep name="define_variables"/>
+        <instructions substitution="false">This step creates zowe-setup-certificates.env and substitutes variables</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>true</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">
+if test ! -d ${USSDIR}; then mkdir -p -m 755 ${USSDIR}; fi
+cp ${zowe_install_path}/bin/zowe-setup-certificates.env ${USSDIR}/zowe-setup-certificates.env
+chmod 755 ${USSDIR}/zowe-setup-certificates.env
+
+HOSTNAME='${HOSTNAME}'
+IPADDRESS='${IPADDRESS}'
+VERIFY_CERTIFICATES='${VERIFY_CERTIFICATES}'
+EXTERNAL_CERTIFICATE='${EXTERNAL_CERTIFICATE}'
+EXTERNAL_CERTIFICATE_ALIAS='${EXTERNAL_CERTIFICATE_ALIAS}'
+EXTERNAL_CERTIFICATE_AUTHORITIES='${EXTERNAL_CERTIFICATE_AUTHORITIES}'
+ZOSMF_CERTIFICATE='${ZOSMF_CERTIFICATE}'
+KEYSTORE_PASSWORD='${KEYSTORE_PASSWORD}'
+KEYSTORE_DIRECTORY='${KEYSTORE_DIRECTORY}'
+KEYSTORE_ALIAS='${KEYSTORE_ALIAS}'
+ZOWE_USER_ID='${ZOWE_USER_ID}'
+ZOWE_GROUP_ID='${ZOWE_GROUP_ID}'
+
+sed -e "s#HOSTNAME=#HOSTNAME=$HOSTNAME#g" -e "s#IPADDRESS=#IPADDRESS=$IPADDRESS#g" -e "s#VERIFY_CERTIFICATES=true#VERIFY_CERTIFICATES=$VERIFY_CERTIFICATES#g" -e "s#EXTERNAL_CERTIFICATE=#EXTERNAL_CERTIFICATE=$EXTERNAL_CERTIFICATE#g" -e "s#EXTERNAL_CERTIFICATE_ALIAS=#EXTERNAL_CERTIFICATE_ALIAS=$EXTERNAL_CERTIFICATE_ALIAS#g" -e "s#EXTERNAL_CERTIFICATE_AUTHORITIES=#EXTERNAL_CERTIFICATE_AUTHORITIES=$EXTERNAL_CERTIFICATE_AUTHORITIES#g" -e "s#ZOSMF_CERTIFICATE=#ZOSMF_CERTIFICATE=$ZOSMF_CERTIFICATE#g" -e "s#KEYSTORE_PASSWORD=password#KEYSTORE_PASSWORD=$KEYSTORE_PASSWORD#g" -e "s#KEYSTORE_DIRECTORY=/global/zowe/keystore#KEYSTORE_DIRECTORY=$KEYSTORE_DIRECTORY#g" -e "s#KEYSTORE_ALIAS=localhost#KEYSTORE_ALIAS=$KEYSTORE_ALIAS#g" -e "s#ZOWE_USER_ID=ZWESVUSR#ZOWE_USER_ID=$ZOWE_USER_ID#g" -e "s#ZOWE_GROUP_ID=ZWEADMIN#ZOWE_GROUP_ID=$ZOWE_GROUP_ID#g" ${zowe_install_path}/bin/zowe-setup-certificates.env > ${USSDIR}/zowe-setup-certificates.env
```

Comment:
I'm not sure if xml has issues with it, but I'd prefer to see this sed command spread over multiple lines, with one substitution per line (followed be \ to indicate continuation) to simplify readability and maintenance.

Reply:
Hi @OnnoVdT ,
I'd prefer that, too, but it is not working on mainframe. The only way I could make this work is like that. :\